tests/acceptance/baseline-generic/new-agent.err

```diff
@@ -3,11 +3,9 @@
 | manage            - not logged in
 === setting up agent configuration
 | using semgrep rules from https://semgrep.dev/c/p/r2c-ci
```

Comment:
Can you run locally and check the logs (in `.semgrep_logs`) just to make sure that the "using semgrepignore" information is still being logged somewhere? Preferably it also logs the "skipping 1 file based on path ignores" information.

Reply:
Yes this gets logged by Semgrep, so it shows up in `.semgrep_logs/semgrep_agent_logs`